### PR TITLE
Fixed a broken job board links `Closes #183`

### LINKED ISSUE #183: [React JS/JavaScript Engineer - Andela] not found in the frontend job board
### Description

The [React JS/JavaScript Engineer - Andela] is not found on the frontend job board. I think the page URL is changed or something, it shows 404. Should I remove it or change it to the correct url?

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by asifurrahaman754**
changed the non-existing job posting link with the actual job board link.

### CODE REVIEW

File: Frontend-Development/Job-Boards/Readme.md [hunk 1]
```diff
@@ -10,5 +10,5 @@
 - [Remote Frontend Developer](https://www.showwcase.com/search?q=front%20end%20developer&tab=jobs) - Showwcase
 - [Web Developer (Frontend)](https://in.indeed.com/jobs?q=frontend&l=remote&vjk=f5117464e8d68692) - Indeed.com
 - [FrontEnd Engineer](https://jobs.lever.co/stackblitz/70ef26ea-cb9d-4184-9f2c-4853078d48f8) - StackBlitz
-- [React JS/JavaScript Engineer](https://remotive.com/remote-jobs/software-dev/reactjs-javascript-engineer-1266003) - Andela
+- [FrotnEnd Developer](https://remotive.com/) - Remotive
```
Comment (siddhant-khisty) [inline Frontend-Development/Job-Boards/Readme.md:13]: It should be `FrontEnd` and not `FrotnEnd`. Please fix the Typo

**Review by siddhant-khisty — COMMENTED** [review]
There seem to be multiple roles here. Please filter it so that only front-end roles are shown. Currently there are no filters and all dev roles are shown

**Comment by asifurrahaman754** [comment]
Thanks, will do more in the future

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

**Pull request merged**